src/bin/tikv-server.rs

```diff
@@ -62,139 +63,108 @@ fn print_usage(program: &str, opts: Options) {
     print!("{}", opts.usage(&brief));
 }
 
-fn get_string_value<F>(short: &str,
-                       long: &str,
-                       matches: &Matches,
-                       config: &toml::Value,
-                       default: Option<String>,
-                       f: F)
-                       -> String
-    where F: Fn(&toml::Value) -> Option<String>
-{
-    // avoid panic if short is not defined.
-    let s = if matches.opt_defined(short) {
-        matches.opt_str(short)
-    } else {
-        None
-    };
+fn exit(code: i32, msg: String) -> ! {
```

Comment:
I see that the code is already 1, why not changing to exit_with_err(msg: String) instead?